renku_notebooks/api/schemas/server_options.py

```diff
@@ -56,6 +56,8 @@ class ServerOptions:
     node_affinities: list[NodeAffinity] = field(default_factory=list)
     tolerations: list[Toleration] = field(default_factory=list)
     resource_class_id: Optional[int] = None
+    idle_threshold: Optional[int] = None
+    hibernation_threshold: Optional[int] = None
```

Comment:
It is more verbose but I would either make this a time delta or add the time unit in the name for the fields.